hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/keys/AddAclKeyHandler.java

```diff
@@ -91,10 +91,15 @@ public Void call() throws Exception {
             OzoneObj.StoreType.valueOf(storeType))
         .build();
 
-    boolean result = client.getObjectStore().addAcl(obj,
+    boolean aclExisted = client.getObjectStore().checkAclExist(obj,
         OzoneAcl.parseAcl(acl));
-
-    System.out.printf("%s%n", "Acl added successfully: " + result);
+    if (aclExisted) {
+      System.out.println("ACL already exists.");
+    } else {
+      boolean result = client.getObjectStore().addAcl(obj,
```

Comment:
I think we don't need a checkAclExist

addAcl semantic true, acl is added successfully, false means acl already exist that is why we cannot add it. So, no need of checkAclExist()

Reply:
Yes, thanks @bharatviswa504 for the comment.

And I found that if the acl already exist when we add it, the ```OMException``` would be thrown by the ```OmOzoneAclMap#addAcl``` result in returning false.
So we don't need ```checkAclExist```.

Fixed.

Reply:
Thanks @bharatviswa504 

Sorry for last comment.
I found that

```ozone sh vol setacl --acls=user:newUser:wr,user:newUser:w,user:newUser:r /vol``` 
will return ```false``` for the already existed acl.

```ozone sh bucket setacl --acls=user:newUser:wr,user:newUser:w,user:newUser:r /vol/buc```
will return ```true```

```ozone sh key setacl --acls=user:newUser:wr,user:newUser:w,user:newUser:r /vol/buc/ikey```
will return ```true```

So I update the ```SetAclVolumeHandler```.

Could we fix the ```SetAclVolumeHandler``` &```SetAclVolumeHandler``` to show the right message for the existed acl ?